src/routing_node.rs

```diff
@@ -14,9 +14,11 @@
 // Software.
 
 extern crate rand;
+extern crate chrono;
```

Comment:
I think the only place needing extern crate is lib.rs here. so this can be replaced with `use rand` and `use chrono` in this case I think anyway.